src/components/common/global/CommonsNavigationButton.vue

```diff
@@ -9,10 +9,22 @@
   >
     <div class="x-button">{{ level }}</div>
     <p class="x-text">{{ levelName }}</p>
+    <modal ref="modal" :bg="false">
+      <div class="modal-content">
+        <h2 class="teko-subtitle">Going back will reset your parameters</h2>
+        <p class="text-center">If you go back the parameters you previously selected will be reset.</p>
+        <button commons class="mt-2" @click="$router.push(`/level/${level}/1`)">Continue</button>
+        <div style="margin-top: 16px;">
+          <p class="x-text-button mh-0p5 inline" @click="clickTest">Cancel</p>
+        </div>
+      </div>
+    </modal>
```

Comment:
Here is the problem with the cancel button. You need to place the `modal` outside of the `grid`, and then wrap both of them with a simple `div`.